strategies/attestationpool/combined/attestationdata_test.go

```diff
@@ -0,0 +1,172 @@
+// Copyright © 2025 Attestant Limited.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package combined_test
+
+// TODO
```

Comment:
I'm guessing you didnt get around to this as it looks like this is a copy of attestationdata_test from the best attestationdata strategy.

Reply:
Now updated.